linode/vpcips/datasource_test.go

```diff
@@ -0,0 +1,71 @@
+//go:build integration || vpc_ips
+
+package vpcips_test
+
+import (
+	"fmt"
+	"log"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/linode/terraform-provider-linode/v2/linode/acceptance"
+	"github.com/linode/terraform-provider-linode/v2/linode/vpcips/tmpl"
+)
+
+func TestAccDataSourceVPCIPs_basic(t *testing.T) {
+	t.Parallel()
+
+	resourceName_all_ips := "data.linode_vpc_ips.foobar"
+	resourceName_vpc_ips := "data.linode_vpc_ips.barfoo"
+	resourceName_filtered_ips := "data.linode_vpc_ips.foobar_filter"
+
+	vpcLabel := acctest.RandomWithPrefix("tf-test")
+	testRegion, err := acceptance.GetRandomRegionWithCaps([]string{"VPCs"})
```

Comment:
```suggestion
	testRegion, err := acceptance.GetRandomRegionWithCaps([]string{"VPCs"}, "core")
```

Don't forget this parameter updated in dev branch so we won't need another PR to fix it lol.
(only apply this change after merging dev to this branch)

Reply:
Good catch! Fixed this.